packages/SwingSet/test/metering/test-dynamic-vat-metered.js

```diff
@@ -91,10 +91,7 @@ test('metering dynamic vats', async t => {
 
   t.deepEqual(
     nextLog(),
-    [
-      'did explode: vat terminated',
-      'terminated: Error: Allocate meter exceeded',
-    ],
+    ['did explode: vat terminated', 'terminated: Allocate meter exceeded'],
```

Comment:
@warner 
I don't see what dropped "Error: " from the message, and I can't tell whether it's an issue. If you don't like the result, let's work on this tomorrow to see how to fix it.